### ISSUE TITLE
How do I make all elements show when minimized?

### ISSUE THREAD

**Issue by da3dsoul1** [tomasklaen/uosc#618]
So via timeline_size_min=30 in uosc.conf I can keep timeline on screen when in windowed mode, how do I make other elements such as track name, controls, etc remain when minimized?

For better visual understanding I made a screenshot with quick drawing of elements I want to appear when in the windowed mode 
![image](https://github.com/tomasklaen/uosc/assets/111184581/ac077317-8f27-49d0-8518-f84c436373ab)

In other words I want to disable proximity mode when minimized
![image](https://github.com/tomasklaen/uosc/assets/111184581/2aa6dcef-6797-43f7-88d2-4197dc4693ec)

**Comment by tomasklaen** [comment]
Currently, you'd have to create a script that sends `set-min-visibility 1/0` message to uosc when mpv switched between windowed and fullscreen.

But I think it wouldn't be a bad idea to add `windowed` and `fullscreen` as element persistency options.

**Comment by da3dsoul1** [comment]
> But I think it wouldn't be a bad idea to add `windowed` and `fullscreen` as element persistency options.

Would be awesome to see it implemented in the main branch

**Comment by natural-harmonia-gropius** [comment]
You can use input.conf

```
KEY           script-message-to uosc set-min-visibility 1
KEY           script-message-to uosc set-min-visibility 0
```

**Comment by da3dsoul1** [comment]
> KEY           script-message-to uosc set-min-visibility 1

This is exactly what I want but only when in windowed mode or when paused, how can I do that?

**Comment by natural-harmonia-gropius** [comment]
> when in windowed mode

Currently can't by using .conf

> when paused

Add `script-opts-append=uosc-top_bar_persistency=paused` and every `*_persistency` options in https://github.com/tomasklaen/uosc/blob/main/script-opts/uosc.conf to your conf

**Comment by da3dsoul1** [comment]
> Add `script-opts-append=uosc-top_bar_persistency=paused` and every `*_persistency` options in https://github.com/tomasklaen/uosc/blob/main/script-opts/uosc.conf to your mpv.conf

Thank you so much, this works like a charm
